app/views/foreman_bootdisk/host.erb

```diff
@@ -14,26 +14,29 @@ bootdisk_raise(N_('Subnet (%s) has no gateway defined'), interface.subnet) if in
 bootdisk_raise(N_('Subnet (%s) has no primary DNS server defined'), interface.subnet) if interface.subnet.dns_primary.nil? || interface.subnet.dns_primary.empty?
 %>
 
+echo Foreman Bootdisk: Host image (<%= @host.name %>)
+
 # loop over net* until the host's MAC matches
 <% (0..32).each do |i| -%>
 :net<%= i %>
 isset ${net<%= i -%>/mac} || goto no_nic
 echo net<%= i -%> is a ${net<%= i -%>/chip} with MAC ${net<%= i -%>/mac}
 iseq ${net<%= i -%>/mac} <%= interface.mac -%> || goto net<%= i+1 %>
-ifopen net<%= i %>
 set idx:int32 <%= i %>
 goto loop_success
 <% end -%>
 
 :loop_success
-echo Configuring net${idx} for static IP address
+echo Configuring net${idx} for static IP address <%= interface.ip %>
 ifopen net${idx}
-set netX/ip <%= interface.ip %>
-set netX/netmask <%= interface.subnet.mask %>
-set netX/gateway <%= interface.subnet.gateway %>
+set net${idx}/ip <%= interface.ip %>
+set net${idx}/netmask <%= interface.subnet.mask %>
+set net${idx}/gateway <%= interface.subnet.gateway %>
+ifstat net${idx}
```

Comment:
Here is the actual typo `netX`. This patch fixes it and also adds `ifstat` - additional info about the configured device.